routes/api/contacts-router.js

```diff
@@ -1,28 +1,90 @@
 import express from "express";
 
+import Joi from "joi";
+
 import contactsService from "../../models/contacts.js";
 
+import { HttpError } from "../../helpers/index.js";
+
 const contactsRouter = express.Router()
 
+const contactAddSchema = Joi.object({
+  name: Joi.string().required().messages({
+    'any.required': `missing required name field`
+  }),
```

Comment:
Schema must be defined in schema.js file

Reply:
create schema.js file